CHANGELOG.md

```diff
@@ -1,15 +1,26 @@
-# [v0.3.1](https://github.com/pybamm-team/BPX/releases/tag/v0.3.1)
-- Temporarily pin Pydantic version ([#35](https://github.com/pybamm-team/BPX/pull/35))
+# [v0.4.0](https://github.com/FaradayInstitution/BPX/releases/tag/v0.4.0)
+
+- Added five parametrisation examples (two DFN parametrisation examples from About:Energy open-source release, blended electrode definition, user-defined 0th-order hysteresis, and SPM parametrisation). ([#45](https://github.com/FaradayInstitution/BPX/pull/45))
+- Allow user-defined parameters to be added using the field ["Parameterisation"]["User-defined"] ([#44](https://github.com/pybamm-team/BPX/pull/44))
```

Comment:
There are still a few links to the old (pybamm-team) repo here and below.

Reply:
thanks, will update